### DISCUSSION TITLE
Release v2.11.0-preview.2

### DISCUSSION THREAD

**Discussion by bruth** [nats-io/nats-server#5336] [General]
## Changelog

*Note: this preview release incorporates the cumulative changes in the 2.10.x series since version 2.10.12.*

### Go Version
- 1.22.2 (#5300)

### Added

TLS
- Add CA search functionality for the Windows cert store (#5115)

JetStream
- Add TPM-driven filestore encryption for Windows (#5273)
- Support `StartTime` for get/direct message requests (#5158)

Monitoring
- Add logical account name to `accstatz` endpoint (#5173)

MQTT
- Add Sparkplug B compatibility (#5150)

WebSocket
- Support custom response headers in configuration (#5230) Thanks to @ramonberrutti for the contribution and @mcosta74 for the proposal!

### Fixed

JetStream
- Fix consumer resume on empty/zero `PauseUntil` value (#5164) Thanks to @MauriceVanVeen for the contribution!

### Complete Changes
 
https://github.com/nats-io/nats-server/compare/v2.11.0-preview.1...v2.11.0-preview.2

<hr /><em>This discussion was created from the release <a href='https://github.com/nats-io/nats-server/releases/tag/v2.11.0-preview.2'>Release v2.11.0-preview.2</a>.</em>

**Comment by alberk8** [comment]
Is there a roadmap for features in V2.11.  I am waiting to test the KV TTL.

**Reply by derekcollison** [reply]
That is coming hopefully in next preview, been caught up on some stability and scale issues with 2.10 series for customers atm.

**Reply by alberk8** [reply]
Are we looking at 2nd or 3rd Quarter for production release?

**Reply by derekcollison** [reply]
We are hoping 2nd.

**Reply by hpvd** [reply]
out of curiosity: we have already seen many very interesting points for 2.11. And time flies...2025...  -> Do you have a tiny but up to date / and of course non-binding hint for a possible release schedule?

(yes, I know you are also hardly working on awesome nex...:-)

**Reply by derekcollison** [reply]
Trying for Feb..

**Reply by hpvd** [reply]
many thanks for this fast and positive answer!
